README.md

```diff
@@ -58,18 +47,21 @@ Decentralized websites using Bitcoin crypto and the BitTorrent network - https:/
 
 #### [More screenshots in ZeroNet docs »](https://zeronet.readthedocs.org/en/latest/using_zeronet/sample_sites/)
 
+Do you want visit sites with .bit extension without localhost address? You can download this extension:
```

Comment:
Maybe also change "without **using a** localhost address?".
Or could just change the whole line to something like: "The following extension converts a localhost address to a .bit extension for easier navigation."